cmd/minishift/cmd/addon/addon.go

```diff
@@ -27,6 +28,10 @@ const (
 	noAddOnMessage  = "No add-on with the name %s is installed."
 )
 
+var (
```

Comment:
I thought this is common to addon and might be used at multiple places. But now I have moved it to `cmd/util/util.go` along with `UnpackAddons`